.github/CODEOWNERS

```diff
@@ -683,17 +683,17 @@
 # TODO(#12694): Replace @nithusha21 as owner after 2021-05-08
 /data/ @kevintab95
 /core/templates/karma.module.ts @vojtechjelinek
-/core/tests/protractor_utils/ @U8NWXD @kevintab95
-/core/tests/protractor_desktop/ @U8NWXD @kevintab95
-/core/tests/protractor/ @U8NWXD @kevintab95
-/core/tests/protractor.conf.js @U8NWXD @kevintab95
-/extensions/interactions/*/protractor.js @U8NWXD @kevintab95
-/extensions/interactions/protractor.js @U8NWXD @kevintab95
-/extensions/rich_text_components/*/protractor.js @U8NWXD @kevintab95
-/extensions/rich_text_components/protractor.js @U8NWXD @kevintab95
-/extensions/objects/protractor.js @U8NWXD @kevintab95
-/scripts/backend_test_shards.json @U8NWXD
-/.github/actions @U8NWXD
+/core/tests/protractor_utils/ @U8NWXD @kevintab95 @krishnarao22
+/core/tests/protractor_desktop/ @U8NWXD @kevintab95 @krishnarao22
+/core/tests/protractor/ @U8NWXD @kevintab95 @krishnarao22
```

Comment:
Are we expecting 3 of the codeowner to approve the PRs? (Just wanted to check the how the review cycle would look like?)

Reply:
No. Any of the 3 codeowners can approve